spec/system/jobseekers_can_sign_up_to_an_account_spec.rb

```diff
@@ -37,41 +37,13 @@
       context "when the confirmation period has expired" do
         before { travel_to 3.hours.from_now }
 
-        it "does not confirm email and redirects to resend confirmation page" do
```

Comment:
Why are you deleting these tests?

Reply:
The logging was put in place by David when it was under investigation.  I removed that code so these events are no longer fired.